src/navigations/HomeBottomTabs.js

```diff
@@ -103,8 +105,17 @@ function HomeBottomTabs() {
   const menuIndicator = (nav, route) => {
     const isAnonChatMenu = route.name === 'AnonymousChannelList';
     const activeColor = isAnonChatMenu ? COLORS.anon_primary : COLORS.blueOnboarding;
-    const style = {backgroundColor: nav.isFocused() ? activeColor : 'transparent'};
-    return <View style={[styles.badge, style]} />;
+    let label = '';
+    if (route.name === 'SignedChannelList') label = 'Convos';
+    else if (route.name === 'AnonymousChannelList') label = 'Incognito';
+    else if (route.name === 'Feed') label = 'Main';
+    else if (route.name === 'News') label = 'News';
+    if (route.name === 'Profile') label = 'Profile';
+
+    const bottomBarExtendedStyle = {
+      color: nav?.isFocused() ? activeColor : COLORS.blackgrey
+    };
+    return <Text style={[styles.bottomBarLabel, bottomBarExtendedStyle]}>{label}</Text>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `menuIndicator` has been significantly refactored. Instead of returning a styled view, it now returns a styled text element with the label of the route. This change improves the user interface by providing more information about each tab.
```diff
- const style = {backgroundColor: nav.isFocused() ? activeColor : 'transparent'};
- return <View style={[styles.badge, style]} />;
+ let label = '';
+ if (route.name === 'SignedChannelList') label = 'Convos';
+ else if (route.name === 'AnonymousChannelList') label = 'Incognito';
+ else if (route.name === 'Feed') label = 'Main';
+ else if (route.name === 'News') label = 'News';
+ if (route.name === 'Profile') label = 'Profile';
+ const bottomBarExtendedStyle = {
+   color: nav?.isFocused() ? activeColor : COLORS.blackgrey
+ };
+ return <Text style={[styles.bottomBarLabel, bottomBarExtendedStyle]}>{label}</Text>;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->